coderd/coderd.go

```diff
@@ -1057,6 +1057,10 @@ func New(options *Options) *API {
 			r.Get("/rich-parameters", api.templateVersionRichParameters)
 			r.Get("/external-auth", api.templateVersionExternalAuth)
 			r.Get("/variables", api.templateVersionVariables)
+			r.Route("/presets", func(r chi.Router) {
```

Comment:
I worry about publishing these separate APIs when we only really need one.
In other words: we can fetch the preset and all its params in one endpoint, rather than splitting them.
I recall we discussed this: did you opt to keep it this way for some reason?

Reply:
> did you opt to keep it this way for some reason?

The conclusion of our discussion was to defer the change.
We weren't clear on until when. Happy to do it now.

Reply:
done.